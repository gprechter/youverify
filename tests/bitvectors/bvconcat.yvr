a: BV[4]
b: BV[4]
c: BV[8]
d: BV[8]
e: BV[6]

a = BV{5, 4}
b = BV{4, 4}
call c = concat(a, b)
call d = concat(b, a)
call e = concat(BV{3, 2}, a)
