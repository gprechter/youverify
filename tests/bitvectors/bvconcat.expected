{
    "a": 5,
    "b": 4,
    "c": 84,
    "d": 69,
    "e": 53
}